README.md

```diff
@@ -26,6 +26,14 @@ We need to keep the GitHub issue tracker clear for scientific collaborators to d
 
 Thanks so much for your help as we deal with the very large influx of interest!
 
+## License
+
+All scripts, datasets, and documentation are [permissively and openly licensed](LICENSE.md) consistent with Victoria Stodden's [Reproducible Research Standard](https://web.stanford.edu/~vcs/talks/VictoriaStoddenCommuniaJune2009-2.pdf) to ensure that they can be maximally reused, modified, and redistributed.
+
+* All **media components** (text, figures, etc.) are licensed under the [Attribution International (CC BY) 4.0 License](https://creativecommons.org/licenses/by/4.0/)
```

Comment:
I just want to check that these licenses are consistent with those associated with the FAH project, it might not be necessary to have these here

Reply:
Ah it's because you've branched from `master`, which is where these licenses were first added so we are all good